codewof/programming/content/en/long-count/question.md

```diff
@@ -0,0 +1,5 @@
+# Long count
+
+Build a function `long_count(list)` that returns the length of `list`.
+
+*The simplest solution isn't an option for this question!*
```

Comment:
This seems ambiguous - I didn't understand the question until I looked at the solution. We also probably shouldn't use `list` as a variable. The wording could be changed to:

Write a function `long_count(list_to_count)` that counts how many items are in `list_to_count`, and then **returns** that count.

*You cannot use the* `len` *function to answer this question*

Reply:
Oh whoops I forgot list is a python thing.
Note that this is *only* a parsons question, I wanted the user to think "oh, that's a len question" and then see that it isn't in the list of lines and try to figure it out another way. Should I abandon the intention?

Reply:
Would this be better?
```
Build a function `long_count(items)` that **returns** the length of the list `items`.

*The simplest solution isn't an option for this question!*

Note that the length of a list is equal to the number of items in it.
```

Reply:
Ah - I didn't notice that it was a parson's question, you are totally right. It is fine to leave it as it is :)